.travis.yml

```diff
@@ -36,6 +36,19 @@ before_install:
   - curl
     https://raw.githubusercontent.com/golang/dep/${DEP_VERSION}/install.sh|sh
 
+before_script:
+  - curl https://download.ceph.com/keys/release.asc | sudo apt-key add -
+  - sudo apt-add-repository
+    "deb https://download.ceph.com/debian-nautilus xenial main"
+  # Ceph does not guarantee arm64 builds, fallback to ceph 14.2.5
+  # yamllint disable rule:line-length
+  - test "$(arch)" != "aarch64" || sudo apt-add-repository
+    "deb https://chacra.ceph.com/r/ceph/nautilus/af06652dc9b2da8c6aadbbecdfafdc7e235abe7d/ubuntu/xenial/flavors/default/ xenial main"
```

Comment:
going forward is this going to be an issue from building with nautilus?

Reply:
ARM builds are currently best effort, and with some releases they are not available. The additional repository contains a build that can be used to fall-back on (got it from the Ceph release maintainers). As ARM systems become more common, I expect to see builds for it improving.
The Ceph team also provides the ceph:ceph base image we consume, that has similar limitations as the builds of the packages.